Tests/Marketplace/marketplace_services.py

```diff
@@ -1229,6 +1231,88 @@ def get_changelog_latest_rn(self, changelog_index_path: str) -> Tuple[dict, Loos
 
         return changelog, changelog_latest_rn_version
 
+    def get_modified_release_notes_lines(self, release_notes_dir: str, changelog_latest_rn_version: LooseVersion,
+                                         changelog: dict, modified_rn_files: list):
+        """
+        In the case where an rn file was changed, this function returns the new content
+        of the release note in the format suitable for the changelog file.
+        In general, if two rn files are created between two consecutive upload runs (i.e. pack was changed twice),
+        the rn files are being aggregated and the latter version is the one that is being used as a key in the changelog
+        file, and the aggregated rns as the value.
+        Hence, in the case of changing an rn as such, this function re-aggregates all of the rns under the
+        corresponding version key, and returns the aggregated data, in the right format, as value under that key.
+
+        Args:
+            release_notes_dir (str): the path to the release notes dir
+            changelog_latest_rn_version (LooseVersion): the last version of release notes in the changelog.json file
+            changelog (dict): the changelog from the production bucket.
+            modified_rn_files (list): a list of the rn files that were modified according to the last commit in
+             'filename.md' format.
+
+        Returns:
+            A dict of modified version and their release notes contents, for modified
+              in the current index file
+
+
+        """
+
+        modified_versions_dict = {}
+
+        for rn_filename in modified_rn_files:
+            _version = rn_filename.replace('.md', '')
+            version = _version.replace('_', '.')
+            # Should only apply on modified files that are not the last rn file
+            if LooseVersion(version) < changelog_latest_rn_version:
```

Comment:
reverse the condition to decrease the "area under the indent"

Reply:
Done